src/App.jsx

```diff
@@ -11,6 +11,8 @@ import Home from "./components/Home";
 
 
 function App() {
+
+  const username= 'jessjelly'
```

Comment:
Have a look at Context to see how you can consume this variably more easily than passing on props.